x-pack/filebeat/docs/inputs/input-httpjson.asciidoc

```diff
@@ -931,6 +932,149 @@ This will output:
 ]
 ----
 
+[[chain]]
+[float]
+=== `chain`
+
+Chain is a list of calls to be made after the first call.
+
+chain list option: [`step`].
+
+[float]
+==== `chain[].step`
+
+chain[].step will contain basic request and response configurations for chain calls.
+
+[float]
+==== `chain[].step.request`
+
+Please refer <<request-parameters,request parameters>>. also place same replace string in url where collected values from previous call should be place. Required. 
+
+Example:
+
+first call: https://example.com/services/data/v1.0/
+
+second call: https://example.com/services/data/v1.0/1/export_ids
+
+third call: https://example.com/services/data/v1.0/export_ids/file_1/info
+
+[float]
+==== `chain[].step.respose.split`
+
+Please refer <<response-split,response split parameter>>. Required.
+
+[float]
+==== `chain[].step.replace`
+
+replace is JSONPath string to parse values from response JSONs, collected from previous chain calls. Please look at the official doc of https://goessner.net/articles/JsonPath/index.html#e2[JSONPath]. Required.
```

Comment:
```suggestion
A [JSONPath](https://goessner.net/articles/JsonPath/index.html#e2[JSONPath]) string to parse values from responses JSON, collected from previous chain steps. Required.
```

Is this always required? Can there be situations were a step does not expect a replaceable part?

Reply:
No, It's always required.